src/pages/CoHarvest/components/BiddingChart/index.tsx

```diff
@@ -57,7 +56,7 @@ const BiddingChart = (props: { round: number; bidInfo }) => {
               key={key}
               data={{
                 percent: e.percentage,
-                volume: formatNumberKMB(getUsd(e.total_bid_amount, ORAIX_TOKEN_INFO, prices)),
+                volume: getUsd(e.total_bid_amount, ORAIX_TOKEN_INFO, prices),
```

Comment:
<img width="902" alt="image" src="https://github.com/oraichain/oraiswap-frontend/assets/60279723/e423a0e4-bc1b-437b-b524-1ed35541ccb9">

cái này phải dùng hàm này nhé để tránh trường hợp volume to